src/MICore/LaunchOptions.xsd

```diff
@@ -630,4 +632,25 @@
       <xs:element name="SourceMapEntry" minOccurs="0" maxOccurs="unbounded" type="SourceMapEntry"/>
     </xs:sequence>
   </xs:complexType>
-</xs:schema>
\ No newline at end of file
+
+  <xs:complexType name="SymbolLoadInfo">
+    <xs:annotation>
+      <xs:documentation>Supports explcit control of symbol loading. The processing of Exceptions lists and symserver entries requires WaitDynamicLoadLib to be true.</xs:documentation>
+    </xs:annotation>
+    <xs:simpleContent>
+      <xs:extension base="xs:string">
+        <xs:attribute name="LoadAll" type="xs:boolean" use="optional">
+          <xs:annotation>
+            <xs:documentation>If true, symbols for all libs will be loaded, otherwise no solib symbols will be loaded. Modified by ExceptionList. Default value is true.</xs:documentation>
+          </xs:annotation>
+        </xs:attribute>
+        <xs:attribute name="ExceptionList" type="xs:string" use="optional">
+          <xs:annotation>
+            <xs:documentation>List of regular expressions. Modifies behavior of LoadAll. If LoadAll is true then don't load symbols for libs that match any regexp in the list. Otherwise only load symbols for libs that match.</xs:documentation>
```

Comment:
>List of regular expressions [](start = 30, length = 27)

You might want to make these wildcards instead of regular expressions. Regular expressions are a little inconvenient to use with file paths or file names since the special RegEx characters are characters that often appear in file names (ex: '.'). Also the VS symbol options uses wildcards instead.

You can implement wildcards by using Regular expressions pretty easily. I will email you a link to the vsdbg code for dealing with include/exclude lists if you like.

Reply:
Now using the vsdbg include/exclude lists. Thanks.